features/fxa-migration/README.md

```diff
@@ -0,0 +1,38 @@
+
+AMO to Firefox Account Migration UX Specs
+=========================================
+
+So that I can reduce the numbers of logins I have, as a Mozilla Add-ons user, I want to switch from using my AMO login and start using a Firefox Account.
+
+## User Flow Diagram
+Red line denotes common post-migration login experience.
+![User Flow Diagram](https://www.lucidchart.com/publicSegments/view/c45f10ff-5084-407d-8322-cbc8b6cd25be/image.png)
+
+## Migrating from AMO
+* When a currently authenticated AMO user returns to AMO, they will see a notice prompting them to upgrade their account.
+* When a currently authenticated AMO user chooses the call to action in the notice, they are taken to the AMO login to confirm their identity.
+* When an existing AMO user enters their email address, they are taken to the AMO login.
+* When they log in successfully, they are shown a notice prompting them to upgrade their account.
+* When they click **Continue** and do not have a Firefox Account, they taken to FxA registration that is pre-populated with their email.
+* When they click **Continue** and do have a Firefox Account, they taken to FxA login that is pre-populated with their email.
```

Comment:
Right, that's the current behaviour, and that's what I meant by "we may have some small amount of work to do on our side".  If you pass in an explicit email address to this flow, I think it would be sensible for us to expand the logic here to e.g. check the server for the existence of an account and show the right form.